Dockerfile

```diff
@@ -48,8 +54,6 @@ rm -rf Slides/.git
 RUN git clone https://github.com/MaRDI4NFDI/WikibaseImport.git WikibaseImport &&\
 rm -rf WikibaseImport/.git
 
-RUN git clone https://github.com/MaRDI4NFDI/mathsearch_extension.git MathSearch &&\
```

Comment:
i would rather have this repository in current deployment to have working state in production until everything is checked on the other repository.